maps-app/build.gradle

```
@@ -9,6 +9,8 @@ android {
         minSdkVersion rootProject.minSdkVersion
         targetSdkVersion rootProject.targetSdkVersion
         testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+        versionCode = 1
+        versionName = '1.0.8'
```

According to [Android's documentation](https://developer.android.com/studio/publish/versioning#appversioning), I don't think we need the `=` signs here:

```suggestion
        versionCode 1
        versionName '1.0.8'
```

---

I'd like to keep as-is. This file was auto-generated by Android Studio.

---

Are you on the latest version of Android Studio ? if not it might be the reason why auto-generation did it that way.

---

Or may be older gradle plugin.

---

When prompted to update, i'm provided this information:

>To take advantage of the latest features, improvements, and security fixes, we strongly recommend that you update the Android Gradle plugin from the current version 3.3.2 to version 3.6.3 and Gradle to version 5.6.4.

Updating now.

---

I've updated Android Studio and gradle, there was no error thrown with keeping `=` as-is. With that said, the Android documentation suggests a format without the `=`. So, i've updated the gradle file accordingly.

https://github.com/Esri/maps-app-android/pull/185/commits/c1b567bd65b0579d57340ddbe1170a333ae21e65